packages/mongodb-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/slice/BuildInfo.kt

```diff
@@ -138,7 +138,7 @@ data class BuildInfo(
                 false,
                 false,
                 null,
-                "",
+                ConnectionString("mongodb://localhost"),
```

Comment:
Just confirming, does it mean that by default the build info will be reported for `mongodb://localhost`. It probably does get overridden when connected, I am still wondering if making this nullable would be better?

Reply:
You are right. nullable would be a good option here for disconnected use cases. Let me change it.

Thanks a lot!